ArchiSteamFarm.CustomPlugins.ExamplePlugin/MeowResponse.cs

```diff
@@ -27,11 +27,19 @@ namespace ArchiSteamFarm.CustomPlugins.ExamplePlugin;
 
 #pragma warning disable CA1812 // False positive, the class is used during json deserialization
 [SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+
 internal sealed class MeowResponse {
-	[JsonProperty("file", Required = Required.Always)]
+	[JsonProperty("id")]
+	internal string? Id { get; set; }
+	[JsonProperty("url", Required = Required.Always)]
 	internal readonly Uri URL = null!;
+	[JsonProperty("width")]
+	internal int? Width { get; set; }
+	[JsonProperty("height")]
+	internal int? Height { get; set; }
```

Comment:
We don't need to declare the properties we are not using.

Reply:
Yes, we don't. I'd like to hear Archi's opinion if we should declare them anyway or not.

Reply:
@Rudokhvist we should remove props we're not using. Less code is better, performance is better, and if we ever need them for whatever reason, we can always bring them back.

Of course in this particular example it doesn't matter since it's just example plugin, but we should aim for best practices and ASF guidelines here as well :+1:

Reply:
I usually remove them in plugins, but since it was an example I wasn't sure. Thanks for reply!